brain_games/scripts/brain_calc.py

```diff
@@ -0,0 +1,53 @@
+from ..cli import welcome_user
+import prompt
+from random import randint, choice
+
+
+name = welcome_user()
+print('What is the result of the expression?')
+
+
+def calculate(number_1, number_2, operation):
+    match operation:
+        case '+':
+            return number_1 + number_2
+        case '-':
+            return number_1 - number_2
+        case _:
```

Comment:
используй константы
```suggestion
        case MATH_ADD:
            return number_1 + number_2
        case MATH_SUB:
            return number_1 - number_2
        case MATH_DIV:
```

обработай дефолтный вариант - выкини ошибку
```
raise Exception(f'Wrong operation "{operation}".  Supported operation - {", ".join(OPERATIONS).}')
```